src/clang-c-frontend/AST/build_ast.cpp

```diff
@@ -27,12 +27,22 @@ static clang::driver::Driver *newDriver(
   const char *BinaryName,
   llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> VFS)
 {
-  clang::driver::Driver *CompilerDriver = new clang::driver::Driver(
+  clang::driver::Driver *CompilerDriver;
+#if CLANG_VERSION_MAJOR >= 13
```

Comment:
Shall we add some comments here to explain this check?

Reply:
We can also bump the minimal version to clang 13.

Reply:
https://github.com/esbmc/esbmc/pull/581/commits/df7913aa29c73b6bb24b54f65c1189e319bfa2d1 was the only other change I needed. It is backwards-compatible. I'll add a note to both places.

Reply:
Done.